openwisp_monitoring/monitoring/apps.py

```diff
@@ -12,5 +15,15 @@ class MonitoringConfig(AppConfig):
 
     def ready(self):
         # create influxdb database if doesn't exist yet
-        create_database()
+        for n in range(5):
+            try:
+                create_database()
+                break
+            except InfluxDBClientError:
+                self.warn_and_delay(n)
```

Comment:
Let's move this block of code to a dedicated method and then call it with `self.create_database()`.
This maintains consistency with other openwisp modules.